forge-gui/res/cardsfolder/upcoming/crucias_titan_of_the_waves.txt

```diff
@@ -0,0 +1,14 @@
+Name:Crucias, Titan of the Waves
+ManaCost:1 B R
+Types:Legendary Creature Human Pirate
+PT:3/3
+T:Mode$ Phase | Phase$ End of Turn | OptionalDecider$ You | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigDiscard | TriggerDescription$ At the beginning of your end step, you may discard a card. If you do, create a Treasure token and choose ambitious or expedient. If you chose ambitious, seek a card with greater mana value than the discarded card. If you chose expedient, seek a card with lesser mana value than the discarded card. 
+SVar:TrigDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose | RememberDiscarded$ True | SubAbility$ DBToken
```

Comment:
make the discard a Cost - then you should be able to use `SVar:X:Discarded$CardManaCost` like Mercurial Chemister

you should also be able to eliminate Remembered/Cleanup line if you do this